agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/e2e/docker-compose-agogosml.yml

```diff
@@ -14,7 +14,7 @@ services:
             EVENT_HUB_NAME: ${EVENT_HUB_NAME_INPUT}
             EVENT_HUB_SAS_POLICY: ${EVENT_HUB_SAS_POLICY}
             LEASE_CONTAINER_NAME: ${LEASE_CONTAINER_NAME_INPUT}
-            APP_HOST: sample
+            APP_HOST: 0.0.0.0
```

Comment:
are you sure about this change? 'sample' is an alias for the sample app, did you run it locally? and the test-gen